ui/package.json

```diff
@@ -47,7 +47,7 @@
     "start": "PORT=3001 craco start --verbose",
     "build": "craco build",
     "test": "craco test",
-    "lint": "eslint --max-warnings 0 src"
+    "lint": "eslint --max-warnings 0 --ext .js,.jsx --fix src"
```

Comment:
What's this for :O

Reply:
* The `--ext` flag is not really necessary, it's just more explicit what kind of files to try to scan.
* The `--fix` flag will auto-format whatever possible: https://masteringjs.io/tutorials/eslint/fix

Reply:
Ahhh I see, thanks for adding them in! I was just wondering what the motivation for adding them was :)